src/auth/interceptors/createRetryInterceptor.js

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+
+const createRetryInterceptor = (options = {}) => {
+  const {
+    httpClient = axios.create(),
+    // This default algorithm is a recreation of what is documented here
+    // https://cloud.google.com/storage/docs/exponential-backoff
+    getBackoffMs = (nthRetry, maximumBackoff = 32000) => {
+      const randomSecond = Math.random();
+      const exponentialBackoff = ((2 ** nthRetry) + randomSecond) * 1000;
```

Comment:
Note: Given I think this had a bug, you may want to consider making `exponentialBackoffMilliseconds` a separate function that could be tested with a couple of options to ensure it returns a result in the expected range.

Reply:
A test would have proven there wasn’t a bug I guess. 😉

Reply:
A test would have proven there wasn’t a bug I guess. 😉

Reply:
A test would have proven there wasn’t a bug I guess. 😉

Reply:
Added! https://github.com/edx/frontend-platform/pull/35/files#diff-154ab735db5f4c582828fb82c237919d